### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by sanscriptum** [BLCM/BLCMods#47]
So far I found only two mods that crash the game on load (namely when you try to start a new game):

*Bessie
*BloodTwitch(hotfix)

My game has all dlcs (except for the last headhunter pack).

**Comment by theNocturni** [comment]
for the Blood Twitch hotfix did you try downloading the newest version?

I recently found out that the 'skill description' part caused the game to crash when loading Krieg
so I took it out
it worked for me after that

**Comment by Bobooo** [comment]
Try using the Bessie mod in the main repository : https://github.com/BL2CP/BLCMods/blob/master/Borderlands%202%20mods/bessie.txt

I actually running this one, and it perfectly works while the one in the MegaCyber folder makes my game crashed.

**Comment by sanscriptum** [comment]
@theNocturni 
Whoops, brainfarted there.
I tried non-hotfix version which I converted into hotfix via converter and it caused the game to crash. Your hotfix version works fine though.

Also whoever uses hotfix mods, you HAVE to put ALL hotfixes under one Transient.SparkServiceConfiguration_6 Keys/Values, because game will activate only one and the last one.

@Bobooo 
Aren't they the same? Both crash the game for me.

**Comment by sanscriptum** [comment]
Also there's an issue with @DaveRabbit's Supremacy, I don't know if it's intentional or not, but its shock damage doesn't scale and is ridiculously high even in normal mode which means it will oneshot almost anyone.

And back to crashing issues: @DaShiku's Ruin smg mod crashed my game when I tried to shoot it when I executed the mod in the main menu and then started to play. When I executed it in the game or just Ruin alone, it worked alright, so I don't know what exactly causes it but that might be some compatibility problem. The only thing I didn't try is to put it back in the inventory and then try to use it, that also might have fix it.
